omnia.sh

```diff
@@ -620,7 +620,12 @@ validate_oim() {
         exit 1
     fi
 
-
+    oim_timezone=$(timedatectl | grep "Time zone" | awk '{print $3}')
+    if [[ -z "$oim_timezone" ]]; then
+        echo -e "${RED}Timezone is not set!${NC}"
```

Comment:
@priti-parate  should we fail here or provide warning and proceed with empty timezone?